wandb/sdk/internal/tb_watcher.py

```diff
@@ -109,11 +111,11 @@ def _calculate_namespace(self, logdir):
             namespace = None
```

Comment:
I think we should remove this heuristic when we have a rootdir that was user specified.